src/app/home/services/events.service.ts

```diff
@@ -8,30 +8,22 @@ import { HttpServiceEvents } from '../../core/services/http-events.service';
 })
 export class EventsService {
 
-    private dataUpdated = new Subject<string>();
+	private searchField = new Subject<string>();
 
-    constructor(private httpServiceEvents: HttpServiceEvents) { }
+	constructor(private httpServiceEvents: HttpServiceEvents) { }
 
-    getDataUpdated() {
-        return this.dataUpdated.asObservable().pipe(
-            startWith(''),
-            debounceTime(1000)
-        );
-    }
+	getSearchStream() {
+		return this.searchField.asObservable().pipe(
```

Comment:
when you use` .pipe` `.asObservable` becomes redundant.
pipe returns Observable interface